pkg/addressbook/addressbook.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package addressbook
+
+import (
+	"github.com/ethersphere/bee/pkg/swarm"
+
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+type GetPutter interface {
```

Comment:
Maybe `GetterPutter`?

Reply:
I believe that GetPutter is analogues to io.ReadWriter in naming. I would rather go with GetPutter, sorry for a late comment.

Reply:
You are right, sorry for confusion 🙈